core/clustersmngr/single_fetcher.go

```diff
@@ -0,0 +1,28 @@
+package clustersmngr
+
+import (
+	"context"
+
+	"k8s.io/client-go/rest"
+)
+
+type singleClusterFetcher struct {
+	restConfig *rest.Config
+}
+
+func NewSingleClusterFetcher(config *rest.Config, _ string) (ClusterFetcher, error) {
```

Comment:
Is that string var meant to be used for something?

Reply:
specify the namespace, it was used for the configmap, and will for the multicluster to fetch the crds

Reply:
done